### PR TITLE
feat: login created and implemented

### PR DESCRIPTION
**Pull request by Rajharsh2**

### CODE REVIEW

File: login-page.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+</head>
+<body>
+    <fieldset>
+        <legend>Login Authentication</legend>
+        <form action="">
+            <br>
+            <label for="mail">Email:</label>
```
Comment (manvithalva12) [inline login-page.html:17]: Raj please explain the procedures using comment line

File: login-page.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
```
Comment (albin1616) [inline login-page.html]: change the title to login page

**Review by albin1616 — CHANGES_REQUESTED** [review]

File: login-page.html (hunk 2 above)
Comment (Rajharsh2) [reply login-page.html]: resolved

File: login-page.html (hunk 1 above)
Comment (Rajharsh2) [reply login-page.html:17]: Resolved

**Review by manvithalva12 — APPROVED** [review]
lgtm

**Review by albin1616 — APPROVED** [review]
lgtm

**Pull request merged**